openbb_sdk/providers/polygon/openbb_polygon/models/stock_news.py

```diff
@@ -93,7 +91,7 @@ def extract_data(
         api_key = credentials.get("polygon_api_key") if credentials else ""
 
         base_url = "https://api.polygon.io/v2/reference/news"
-        querystring = get_querystring(query.dict(by_alias=True), [])
+        querystring = get_querystring(query.model_dump(), [])
```

Comment:
I was checking these cases where `query.model_dump()`, don't we need to keep the `by_alias=True` to send the query fields with the alias instead of the field name?